populus/project.py

```diff
@@ -333,6 +339,16 @@ def get_chain(self, chain_name, chain_config=None):
         chain = chain_config.get_chain(self, chain_name)
         return chain
 
+    def get_web3http_chain(self, rpc_path):
```

Comment:
I don't think these are good APIs.  We already have a way for users to configure web3 via the configuration.  All of this can be accomplished simply with an `ExternalChain` that has the appropriate `web3` configuration under it.